src/content_scripts/prevent-indexeddb-tracking/ghostery-prevent-indexeddb-tracking.js

```diff
@@ -0,0 +1,174 @@
+/**
+ * Ghostery Browser Extension
+ * https://www.ghostery.com/
+ * https://www.whotracks.me/
+ *
+ * Copyright 2017-present Ghostery GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+
+(function () {
+  // 'disable': remove the API (it does not exist on Firefox, so browser-independent code should be able to handle it)
+  // 'hide-all': implement the API but always return an empty list (prevents tracking, but the side-effects are not clear)
+  // 'hide-well-known-databases': like hide-all, but only hide some well-known databases (maybe more predicable, but may still break and still allow tracking)
+  // 'try-to-open': (does not work) try to open the database and skip the ones that are not readable
+  //                (assumes that WebKit rejects it if you try to open a databases cross-origin, but that is not the case)
+  const METHOD = 'hide-well-known-databases';
+
+  if (METHOD === 'disable') {
+    console.log('Monkey patching: remove indexedDB.databases function');
```

Comment:
Guess we have to remove all console logs  - or at least make them `debug` with a appropriate prefix